### ISSUE TITLE
Ability to hide persistent Notification below the fold

### ISSUE THREAD

**Issue by alexbarnsley** [transistorsoft/cordova-background-geolocation-lt#389]
## Your Environment
* Plugin version: 2.7.3
* Platform: Android
* OS version: 7.1.1
* Device manufacturer / model: OnePlus 3
* Cordova version (`cordova -v`): 6.5.0
* Cordova platform version (`cordova platform ls`): 6.0.0
* Plugin config:

```
desiredAccuracy: 0,
distanceFilter: 10,
stationaryRadius: 25,
locationUpdateInterval: 10000,
fastestLocationUpdateInterval: 5000,
activityType: 'AutomotiveNavigation',
activityRecognitionInterval: 10000,
stopTimeout: 5,
stopOnTerminate: false,
startOnBoot: true,
preventSuspend: true,
heartbeatInterval: 60,
foregroundService: true,
notificationTitle: "blaa",
notificationText: "blaa",
```

## Context
When setting `foregroundService` to true, the persistent notification appears at the top, which I understand. However, it is possible to hide persistent notifications so they can only be seen once swiping down on the status bar (the [Twilight App](https://play.google.com/store/apps/details?id=com.urbandroid.lux&hl=en_GB) does this). This way it's less intrusive but the app also doesn't get killed.

I believe the option to do this would be along the lines of `setPriority(Notification.PRIORITY_MIN)` (if this could be done in config, that would be great), which would be an enhancement in this case.

**Comment by christocracy** [comment]
I don't have a problem with this.  Currently no priority was being set.

This will be available in the premium version immediately.  It don't be available here in the public version until the *latest stable release* is next pushed.

**Comment by alexbarnsley** [comment]
That's brilliant, thank you for the prompt reply!

**Comment by christocracy** [comment]
This is now released in `2.7.4`
